components/dropDown/dropDown.js

```diff
@@ -83,29 +83,46 @@ class DropDown extends Component {
   }
 
   render() {
+    const {
+      clearable,
+      dataAttrs = {},
+      filterMode,
+      mods = [],
+      multi,
+      name,
+      options,
+      placeholder,
+      scrollMenuIntoView,
+      searchable,
+      value,
+      ...otherProps
+    } = this.props;
+
     const isFiltermode = this.props.filterMode;
-    const mods = {
-      'filter': isFiltermode,
-      'state-active': isFiltermode && this.props.options.some(option => option.checked),
-    };
+
+    filterMode && mods.push('filter');
+    filterMode && this.props.options.some(option => option.checked) && mods.push('state-active');
 
     const className = getClassNamesWithMods('ui-dropdown', mods);
 
     return (
-      <Select
-        className={className}
-        clearable={this.props.clearable}
-        menuRenderer={this.menuRenderer}
-        multi={isFiltermode ? true : this.props.multi}
-        name={this.props.name}
-        onChange={this.onChange}
-        optionComponent={isFiltermode ? DropdownFilterOptionComponent : undefined}
-        options={this.props.options}
-        placeholder={this.props.placeholder}
-        scrollMenuIntoView={this.props.scrollMenuIntoView}
-        searchable={this.props.searchable}
-        value={this.props.value}
-      />
+      <div {...getDataAttributes(dataAttrs)}>
+        <Select
```

Comment:
The Select component can handle data attributes fine, via inputProps. No need for wrapping.

    <Select inputProps={getDataAttributes(dataAttrs)}>